gdal/ogr/ogrsf_frmts/s57/s57reader.cpp

```diff
@@ -1506,7 +1506,30 @@ OGRFeature *S57Reader::ReadVector( int nFeatureId, int nRCNM )
                              poRecord->GetIntSubfield("VRPT",iField,
                              "MASK",iSubField) );
     }
-
+	
+/* -------------------------------------------------------------------- */
+/*      Geometric attributes                                            */
+/*      Retrieve POSACC and QUAPOS attributes                           */
+/* -------------------------------------------------------------------- */
+
+	const int posaccField = poRegistrar->FindAttrByAcronym("POSACC");
+	const int quaposField = poRegistrar->FindAttrByAcronym("QUAPOS");
+	
+	DDFField * field = poRecord->GetField(i);
```

Comment:
I forgot a line... I checked again the compilation in the GDAL vagrant, it's ok now